src/index.js

```diff
@@ -0,0 +1,7 @@
+import joinAdInterestGroup from './interest-groups/join';
+import leaveAdInterestGroup from './interest-groups/leave';
+
+export default {
+	joinAdInterestGroup,
+	leaveAdInterestGroup,
+};
```

Comment:
@brodrigu this will allow us to expose members, but it won't explicitly require that a consumer use a namespace, such as `fledge.joinAdInterestGroup`.  Do you think we need to require that? Or will it be enough to show it used as such in the README?